chat-plugins/animesearch.js

```diff
@@ -26,12 +27,13 @@ function search (query) {
 		request(link, function (err, response, data) {
 			if (err || response.statusCode !== 200) return reject('Anime results for "' + query + '" were not found...');
 			data = JSON.parse(data);
-			//Uses exact match if found, uses first match otherwise
+			//displays exact match if found, displays first match otherwise
 			let firstMatch, exactMatch;
 			for (let i in data.search) {
 				let info = data.search[i];
 				if (info.type === 'anime') {
-					if (info.link === formattedQuery || format(info.title) === formattedQuery) {
+					console.log(format(info.title));
```

Comment:
Uh, no pls...

Let's not siliently log this kind of thing in the console logs :-1: